src/front-end/src/components/MenteeForm/MenteeForm.test.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import userEvent from '@testing-library/user-event';
+import { act } from 'react-dom/test-utils';
+import { mockWindowMatchMedia } from '../../helper-methods';
+
+import MenteeForm from './MenteeForm';
+
+let container = null;
+
+mockWindowMatchMedia();
+
+beforeEach(() => {
+  container = document.createElement('div');
+  document.body.appendChild(container);
+});
+
+afterEach(() => {
+  // cleanup on exiting
+  unmountComponentAtNode(container);
+  container.remove();
+  container = null;
+});
+
+it('renders without crashing', () => {
+  render(<MenteeForm />, container);
+  expect(container.textContent).toBe(
+    'Sign up as a MenteeNameEmailTime-zoneSelect timezone..Top 3 skills\xa0DescriptionSubmit',
+  );
+});
+
+test('renders the correct header text', () => {
+  render(<MenteeForm />, container);
+  const h1 = document.getElementsByTagName('h1');
+  expect(h1[0].textContent).toBe('Sign up as a Mentee');
+});
+
+test('renders the correct input labels', () => {
+  render(<MenteeForm />, container);
+  const getLabels = document.getElementsByTagName('label');
+  const correctLabels = [
+    'Name',
+    'Email',
+    'Time-zone',
+    'Top 3 skills',
+    'Description',
+  ];
+  let i = 0;
+  while (i < getLabels.length) {
+    expect(getLabels[i].textContent).toBe(correctLabels[i]);
+    i += 1;
+  }
```

Comment:
Why not a for loop? I'd change this to 
```suggestion
  for (let i = 0; i < getLabels.length; i += 1) {
    expect(getLabels[i].textContent).toBe(correctLabels[i]);
  }
```

Reply:
Oh man. It *somehow* completely slipped my mind that in a for loop you can write 'i += 1'. I'm so used to ++ notation, that when I received the prettier warning that ++ notation isn't accepted, my brain decided that the while loop was the way to go. Whoops.